tests/components/freebox/test_device_tracker.py

```diff
@@ -0,0 +1,29 @@
+"""Tests for the Freebox sensors."""
+from unittest.mock import Mock
+
+from freezegun.api import FrozenDateTimeFactory
+
+from homeassistant.components.freebox import SCAN_INTERVAL
+from homeassistant.components.sensor import DOMAIN as SENSOR_DOMAIN
+from homeassistant.core import HomeAssistant
+
+from .common import setup_platform
+
+from tests.common import async_fire_time_changed
+
+
```

Comment:
Add a `async def test_access_point_mode(` test for classic usage

devices count should be getter than 1

Reply:
Import already there